sentences/sl/light_HassLightSet.yaml

```diff
@@ -2,4 +2,38 @@ language: sl
 intents:
   HassLightSet:
     data:
-      - sentences: []
+      - sentences:
+          - "<spremeni> <name> na <brightness>"
+          - "<spremeni> [osvetlitev|jakost|svetlost] [luči|svetila|razsvetljave|svetilke] [v|na] <name> na <brightness> [moči|jakosti|svetlosti|odstotkov|procentov]"
+        response: brightness
+
+      - sentences:
+          - "<spremeni> [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] [v|na] <area> [na] <brightness> [moči|jakosti|svetlosti|odstotkov|procentov]"
+        response: brightness
+
+      #brightness min/max
+      - sentences:
+          - "<spremeni> <name> na {brightness_level:brightness}"
+          - "<spremeni> [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] [v|na] <name> na {brightness_level:brightness} [moči|jakosti|svetlosti|odstotkov|procentov]"
+        slots:
+          brightness: 100
+        response: brightness
+
+      - sentences:
+          - "<spremeni> [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] [v|na] <area> [na] {brightness_level:brightness} [moči|jakosti|svetlosti|odstotkov|procentov]"
```

Comment:
This shouldn't be optional [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] - change to round brackets

Reply:
Changed line 23 - in brackets. Thanks!